docs/winning.rst

```diff
@@ -117,7 +117,7 @@ Writing your own mixins
 You can write your own assertions and mix them in to your test cases, too. We recommend reading the :mod:`marbles.mixins` source code to see how to do this. Here is the :class:`~marbles.mixins.UniqueMixins` source as an example:
 
 .. literalinclude:: ../marbles/mixins/marbles/mixins/mixins.py
-   :lines: 348-425
+   :lines: 355-432
```

Comment:
This is kind of annoying to have to remember to do. One way around always having to remember to do this would be to have one module per mixin class. A better option would be to `literalinclude` parts of the code by reference (class name, method name, etc.). Is that possible? My (very cursory) google search didn't turn up anything.

Reply:
Yes, `literalinclude` has an option `:pyobject:` that does this. It’s very easy to skim past in the Sphinx docs.